File: charactermanager/backend/src/main/java/ai/ragu/charactermanager/pojo/AncestryAbilityBoosts.java
package ai.ragu.charactermanager.pojo;

import org.hibernate.annotations.Table;

import javax.persistence.Entity;
import javax.persistence.ForeignKey;
import javax.persistence.ManyToOne;

@Entity
@Table(appliesTo = "AncestryAbilityBoosts")
public class AncestryAbilityBoosts {


    @ManyToOne
    @ForeignKey(name = "FK_COUNTRY")
    private String ancestryId;

    @ManyToOne
    @ForeignKey(name = "FK_COUNTRY")
    private String abilityScoreId;

}

File: charactermanager/backend/src/main/java/ai/ragu/charactermanager/pojo/AncestryAbilityBoosts.kt
package ai.ragu.charactermanager.pojo

import AbstractJpaPersistable
import javax.persistence.Entity
import javax.persistence.ManyToOne

@Entity
class AncestryAbilityBoosts(
        @ManyToOne
        val ancestryId: String? = null,

        @ManyToOne
        val abilityScoreId: String? = null
) : AbstractJpaPersistable<Long>()